app/presenters/backup_code_reminder_presenter.rb

```diff
@@ -0,0 +1,21 @@
+class BackupCodeReminderPresenter
```

Comment:
What do you think of dropping the presenter and putting the `I18n` calls directly in the template?